pagopa/pn-frontend-e2e
# Pn-10249-10251-10252-10253

## Short description

Tests 10249, 10251, 10252, 10253

## List of changes proposed in this pull request

- 10249: check if the delegato sees the cancelled notification sent to delegante
   - added step to check the delegante's notification section
- 10251: the notification is cancelled just after it is sent to the courtesy contact (mail)
   - added step to check when the notification is sent to the courtesy contact
- 10252: the notification is cancelled just after the PEC is sent (mail)
   - added step to check when the notification is sent to the PEC
-10253

## How to test

Simply run 
`mvn clean test -D"cucumber.filter.tags=@TA_annullamentoNotificaMittenteEControlloDownloadPEC"`  10253
`mvn clean test -D"cucumber.filter.tags=@TA_DelegvisualizzazioneDettaglioNotifichaAnnullata"`  10249
`mvn clean test -D"cucumber.filter.tags=@TA_annullamentoNotificaMittenteEControlloPEC"`  10252
`mvn clean test -D"cucumber.filter.tags=@TA_annullamentoNotificaMittenteEControlloMailDiCortesia"`  10251



and verify the tests work fine
